### ISSUE TITLE
!!!The rotate artboard command does not work.

### ISSUE THREAD

**Issue by disdontwork** [MengTo/Angle-Sketch-Plugin#61]
This is pretty urgent and kinda breaks the workflow, when can we expect this to be fixed?

**Comment by aggerbo** [comment]
I agree, major issue. I'm currently generating 10+ screenshots for 4 different languages. So I rely on this plugin for 40 screenshots and I'm loosing my mind.

**Comment by dbrienswann** [comment]
Rotate still missing. What is the deal?
